pkg/bootstrap/testdata/all_golden.json

```diff
@@ -358,8 +360,8 @@
               ],
               "validation_context": {
                 "trusted_ca": {
-                  "filename": "./var/run/secrets/istio/root-cert.pem"
-                },
+                  "filename": ""
```

Comment:
what is triggering this to be empty? Bad test or bad code? If we actually generate this it will block startup (I think its invalid config)

Reply:
The template is tested without setting the parameter - that will be done in agent as part of the init, before calling the template.

I can change the code to set something.